spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/LibraryControllerHelperService.java

```diff
@@ -359,164 +329,6 @@ public JSONObject changeLibraryIdBarcode(HttpSession session, JSONObject json) {
     return JSONUtils.SimpleJSONResponse("New identification barcode successfully assigned.");
   }
 
-  public JSONObject getLibraryDilutionBarcode(HttpSession session, JSONObject json) {
```

Comment:
I'd like to have the print barcodes on everything. I'll put that on my short list.